manager/controllers/baseline_create.go

```diff
@@ -58,6 +58,10 @@ func CreateBaselineHandler(c *gin.Context) {
 		return
 	}
 
+	if request.Description != nil && *request.Description == "" {
+		request.Description = nil
+	}
```

Comment:
There's EmptyToNil() function for this.
And you can actually fix it because it does not check if `s != nil`. :-)